src/framework/ui/uiwidgettext.cpp

```diff
@@ -56,6 +56,15 @@ void UIWidget::updateText()
     m_textMustRecache = true;
 }
 
+void UIWidget::resizeToText()
+{
+    auto textSize = getTextSize();
+    textSize += Size(m_padding.left + m_padding.right, m_padding.top + m_padding.bottom);
+    textSize += m_textOffset.toSize();
+
+    setSize(textSize);
```

Comment:
```suggestion
void UIWidget::resizeToText()
{
    setSize(getTextSize() + Size(m_padding.left + m_padding.right, m_padding.top + m_padding.bottom) + m_textOffset.toSize());
```

Is the temp var needed?

Reply:
The temporary is used for the sake of readability, it will be optimized away anyway.